plow/bexp/utils.flow

```diff
@@ -134,10 +136,10 @@ extractTypeNamesBType(acc : Set<string>, t : BType) -> Set<string> {
 	switch (t) {
 		BTypePar(id): acc;
 		BTypeName(id, typars): {
-			acc1 = if (startsWith(id, "?")) acc else insertSet(acc, id);
```

Comment:
Are you sure you do not want to filter out the ? names?

Reply:
That was an occasional bug, thanks for noticing it